lib/metadata/metadataUtils.js

```diff
@@ -84,6 +89,40 @@ function metadataGetObject(bucketName, objectKey, versionId, log, cb) {
         });
 }
 
+/** metadataGetObjects - retrieves specified object or version from metadata. This
+ * method uses cursors, hence is only compatible with a MongoDB DB backend.
+ * @param {string} bucketName - name of bucket
+ * @param {string} objectsKeys - name of object key
+ * @param {RequestLogger} log - request logger
+ * @param {function} cb - callback
+ * @return {undefined} - and call callback with err, bucket md and object md
+ */
+function metadataGetObjects(bucketName, objectsKeys, log, cb) {
+    const options = { getDeleteMarker: true };
+    const objects = objectsKeys.map(objectKey => ({
+        key: objectKey ? objectKey.inPlay.key : null,
+        params: options,
+        versionId: objectKey ? objectKey.versionId : null,
+    }));
+
+    // Returned objects are following the following format: { key, doc, versionId }
+    // That is required with batching to properly map the objects
+    return metadata.getObjectsMD(bucketName, objects, log, (err, objMds) => {
+        if (err) {
+            log.debug('err getting object MD from metadata', { error: err });
```

Comment:
```suggestion
            log.debug('error getting object MD from metadata', { error: err });
```

Reply:
Adding a 's' also to 'object' to differentiate with the other method.